iina/VideoView.swift

```diff
@@ -400,5 +405,16 @@ class VideoView: NSOpenGLView {
   func ignoreGLError() {
     glGetError()
   }
-
+  
+  override func draggingEntered(_ sender: NSDraggingInfo) -> NSDragOperation {
+    return .copy
+  }
+  
+  override func performDragOperation(_ sender: NSDraggingInfo) -> Bool {
+    let board = sender.draggingPasteboard()
+    let fileURL: NSURL? = NSURL(from: board)
+    playerCore.loadExternalSubFile(fileURL as! URL)
```

Comment:
Do you have a list of the support format of subtitles, audio tracks and videos? I think it helps to handle.